.travis.yml

```diff
@@ -14,7 +14,12 @@ cache:
   directories:
     - $HOME/.m2
 
-script: ./mvnw clean verify
+env:
+  matrix:
+    -
+    - PROFILE='-P spring5'
```

Comment:
Interesting fact: The `spring4-junit5` profile is active by default and it will be disabled as soon as we enable `spring5`. I tried the same in logbook and tracer but there it doesn't work because the `spring5` profile is defined in the parent, but the `spring4-junit5` in a module. The docs are subtle, but they have a very important sentence about this:

> This profile will automatically be active for all builds unless another profile **in the same POM** is activated using one of the previously described methods.
>
> http://maven.apache.org/guides/introduction/introduction-to-profiles.html

Really unexpected, I'd say.